src/Scoringline/SendinblueApi/Api/Email.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+/**
+ * This file is a part of scoringline sendinblue api package
+ *
+ * (c) Scoringline <m.veber@scoringline.com>
+ *
+ * @author Joni Rajput <joni@sendinblue.com>
+ *
+ * For the full license, take a look to the LICENSE file
+ * on the root directory of this project
+ */
+namespace Scoringline\SendinblueApi\Api;
+
+use Nekland\BaseApi\Api\AbstractApi;
+use Scoringline\SendinblueApi\Exception\EmailSendFailureException;
+use Scoringline\SendinblueApi\Model\Email as EmailModel;
+
+class Email extends AbstractApi
+{
+    const API_URL = '/email';
+
+    /**
+     * @param EmailModel $emailModel
+     * @param string $type
+     * @return array
+     * @throws \Exception
+     */
+    public function sendEmail(EmailModel $emailModel, $type = 'basic')
+    {
+        if($type === 'advance') {
+            $result = $this->post(self::API_URL, json_encode($this->getAdvanceParams($emailModel)));
+        } else {
+            $result = $this->post(self::API_URL, json_encode($this->getBasicParams($emailModel)));
+        }
+
+        if ($result['code'] === 'failure') {
+            throw new EmailSendFailureException($result['message']);
+        }
+
+        return $result;
+    }
+
+    /**
+     * @param EmailModel $emailModel
+     * @return array
+     */
+    private function getBasicParams(EmailModel $emailModel)
+    {
+        $basicParams = [];
+
+        // Set to
+        if (count($emailModel->getTo())) {
+            $basicParams['to'] = $emailModel->getTo();
+        }
+        // Set from
+        if (count($emailModel->getFrom())) {
+            $basicParams['from'] = $emailModel->getFrom();
+        }
+        // Set subject
+        if ($emailModel->getSubject()) {
+            $basicParams['subject'] = $emailModel->getSubject();
+        }
+        // Set text
+        if ($emailModel->getText()) {
+            $basicParams['subject'] = $emailModel->getText();
+        }
+        // Set html
+        if ($emailModel->getHtml()) {
+            $basicParams['html'] = $emailModel->getHtml();
+        }
+
+        return $basicParams;
+    }
+
+    /**
+     * @param EmailModel $emailModel
+     * @return array
+     */
+    private function getAdvanceParams(EmailModel $emailModel)
+    {
+        $advanceParams = [];
+        // SET headers
+        if ($emailModel->getHeaders()) {
+            $advanceParams['headers'] = $emailModel->getHeaders();
+        }
+        // SET replyTo
+        if ($emailModel->getReplyTo()) {
+            $advanceParams['replyto'] = $emailModel->getReplyTo();
+        }
+        // SET cc
+        if ($emailModel->getCc()) {
```

Comment:
I think Sendinblue team test these things too but I added this for more security so that we sent data if really we have. But if not needed then I can remove.